lib/src/resource.dart

```diff
@@ -88,7 +88,7 @@ abstract class Resource<T1> {
 
 /// A wrapper over the data returned by the mobile backend. Signifies a successful
 /// scenario.
-class ResourceSuccess<T1> extends Resource<T1> {
+class ResourceSuccess<T1> extends PeppermintResource<T1> {
```

Comment:
Rename this too and all class in this file

Reply:
okay